LogMonitor/README.md

```diff
@@ -5,14 +5,12 @@
 **Log Monitor** is a log tool for Windows Containers. It monitors configured log sources and pipes a formatted output to a configured output location. Supported log sources include:
 
 - Event Logs
-
 - ETW Providers
-
 - Custom App logs
 
 Supported output locations include:
 
-- STDOUT
+- `STDOUT`
 
 Log Monitor is configured via the Log Monitor Config json file.
 

```

Comment:
Add a link to the latest release: https://github.com/microsoft/windows-container-tools/releases/latest